engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.controller.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.capsule-content.controller:CapsuleContentController
+ *
+ * @requires $scope
+ * @requires $urlMatcherFactory
+ * @requires $location
+ * @requires $translate
+ * @requires $CapsuleContent
+ * @requires $AggregateTask
+ *
+ * @description
+ *   Provides the functionality for the capsule-content page.
+ */
+angular.module('Bastion.capsule-content').controller('CapsuleContentController',
+    ['$scope', '$urlMatcherFactory', '$location', 'translate', 'CapsuleContent', 'AggregateTask',
+    function ($scope, $urlMatcherFactory, $location, translate, CapsuleContent, AggregateTask) {
+
+        var urlMatcher = $urlMatcherFactory.compile("/smart_proxies/:capsuleId");
+        var capsuleId = urlMatcher.exec($location.path()).capsuleId;
```

Comment:
I am assuming this is due to not using states and thus can't rely on `$stateParams` ?